packages/nimble-components/src/rich-text/viewer/index.ts

```diff
@@ -0,0 +1,75 @@
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { observable } from '@microsoft/fast-element';
+import { template } from './template';
+import { styles } from './styles';
+import { RichTextMarkdownParser } from '../models/markdown-parser';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-rich-text-viewer': RichTextViewer;
+    }
+}
+
+/**
+ * A nimble styled rich text viewer
+ */
+export class RichTextViewer extends FoundationElement {
+    /**
+     *
+     * @public
+     * Markdown string to render its corresponding rich text content in the component.
+     */
+    @observable
+    public markdown = '';
+
+    /**
+     * @internal
+     */
+    public viewer!: HTMLDivElement;
+
+    private readonly markdownParser: RichTextMarkdownParser;
+
+    public constructor() {
+        super();
+        this.markdownParser = new RichTextMarkdownParser();
```

Comment:
Can we consider initializing this similar to `rich-text-editor` to maintain consistency?

Reply:
Yes updated as suggested.